roles/openshift_facts/tasks/main.yml

```diff
@@ -7,8 +7,16 @@
 # Locally setup containerized facts for now
 - set_fact:
     l_is_atomic: "{{ ostree_booted.stat.exists }}"
+
+- set_fact:
+    l_is_system_containers: "{{ use_system_containers | default(false) }}"
```

Comment:
Any reason not to add `use_system_containers: false` to defaults/main.yml and then just use `use_system_containers` below?